db/re/skill_db.yml

```diff
@@ -18706,7 +18704,7 @@ Body:
         Area: 6
       - Level: 5
         Area: 7
-    Duration1: 15000
+    Duration1: 18000
```

Comment:
Note: For all other skills you assumed the current value on rA is already correc,t but for this one you decided to change it to its default value. This is just a hint as I don't know much about renewal skills or where their durations come from.

(You also did it for Whiteimprison but our tests clearly showed that the base duration is 18+2, so that's correct.)

Reply:
Confirmed that it is indeed 15 seconds.